.changeset/tasty-planets-buy.md

```diff
@@ -0,0 +1,5 @@
+---
+"@ponder/core": patch
+---
+
+use numeric32 for bigint on postgres instead of bytea
```

Comment:
"Updated Postgres bigint columns to use `numeric32` rather than a custom byte encoding. This makes it easier to run queries directly against the indexed entity tables."

Reply:
Yeah this should be numeric78 instead of 32. I initially used numeric32 because I thought you specified the number of bytes but that's not true. It's the number of digits.